### DISCUSSION TITLE
基于结构化数据的中文语料训练

### DISCUSSION THREAD

**Discussion by AceyKubbo** [ymcui/Chinese-LLaMA-Alpaca#753] [Ideas]
请问大佬们,将来会有打算基于结构化数据(比如表格统计,内容提取等),这种精细化的语义理解展开训练
举个列子
Table:
| 日期 | 来访者 | 访问意图 | 去往楼层 | 
| --- | --- | --- | --- | 
| 2023年7月10日 | 张三 | 商务会议 | 11楼 |
| 2023年7月8日 | 李四 | 推销产品 | 6楼 | 
| 2023年7月8日 | 王五 | 面试 | 5楼 | 
| 2023年7月6日 | 杰克 | 面试 | 5楼 | 

Question:
2023年7月8号有哪些人来访过?
Answer:
来访者有李四和王五

```
Table:
| 日期 | 来访者 | 访问意图 | 去往楼层 | 
| --- | --- | --- | --- | 
| 2023年7月10日 | 张三 | 商务会议 | 11楼 |
| 2023年7月8日 | 李四 | 推销产品 | 6楼 | 
| 2023年7月8日 | 王五 | 面试 | 5楼 | 
| 2023年7月6日 | 杰克 | 面试 | 5楼 | 

Question:
2023年7月8号有哪些人来访过?
Answer:
来访者有李四和王五
```

**Comment by ymcui** [comment]
暂时没有特别的计划。目前模型本身是支持一些简单的结构化数据理解能力的。
以下是Alpaca-Plus-7B（q6_k量化）的一组测试：
![WX20230718-083153@2x](https://github.com/ymcui/Chinese-LLaMA-Alpaca/assets/16095339/7cd92a98-ddb1-4f7b-b8f6-d6bec2472fa7)

**Reply by AceyKubbo** [reply]
那目前的做法只能先把复杂表格拆分成简单表格进行语义分析了
